test/nadia_test.exs

```diff
@@ -267,4 +267,14 @@ defmodule NadiaTest do
       assert :ok = Nadia.delete_sticker_from_set("CAADBQADLgADmEjsA7jm5QOy8WxsAg")
     end
   end
+
+  test "send_animation" do
+    use_cassette "send_animation" do
+      file_id = "BQADBAADhQEAArKi5FCHKeOFAAEsKQkWBA"
+      {:ok, message} = Nadia.send_animation(-261_856_796, file_id)
```

Comment:
looks like a real `chat_id` here, I think it's better to avoid having sensitive data in the repo.
`chat_id` in cassettes will be replaced by `666`, but you have to change it in test files by yourself.

Reply:
@zhyu good catch!